src/components/ToggleButtonGroup/ToggleButton.module.css

```
@@ -0,0 +1,35 @@
+.toggle-button{
+  --toggle-button-label-font-size: 16px;
+  --toggle-button-padding-left-right: 10px;
+  --toggle-button-padding-top-bottom: 7px;
+  --toggle-button-border-radius: 0px;
+  --toggle-button-border: none;
+  --toggle-button-min-width: 125px;
```

We do have a min-width button variable as well in our tokens, but I'm not sure if thats something we want to keep. We have an ongoing dicussion about it, but in the meantime I think you can use it.
```suggestion
  --toggle-button-min-width: var(--component-button-size-min-width);
```